api_v3/services/LiveStreamService.php

```diff
@@ -575,5 +574,97 @@ public function regenerateStreamTokenAction($entryId)
 		$entry = KalturaEntryFactory::getInstanceByType($liveEntry->getType());
 		$entry->fromObject($liveEntry, $this->getResponseProfile());
 		return $entry;
+
 	}
+
+    /**
+     *
+     * @action archive
+     * @param string $liveEntryId
+     * @return bool
+     * @throws KalturaAPIException
+     * @throws KalturaClientException
+     * @throws PropelException
+     */
+	public function archiveAction($liveEntryId)
+    {
+        /** @var LiveStreamEntry $liveEntry */
+        $liveEntry = entryPeer::retrieveByPK($liveEntryId);
+        if (!$liveEntry)
+        {
+            KalturaLog::err("Live entry id [" . $liveEntryId . "] not found");
+            return null;
+        }
+        $cuePoints = CuePointPeer::retrieveByEntryId($liveEntryId);
+
+        $importantCuePointTags = ["entry-reset-mode", "webcast:internal-state", "player-qna-settings-update",
+            "select-deck-document", "poll-data", "timeline-assets-state", "rtc-settings"];
+        /** @var CuePoint $cuePoint */
+
+        foreach ($cuePoints as $cuePoint)
+        {
+            KalturaLog::info("Iterating over cue point [" . $cuePoint->getId() . "]");
+            $cuePointTags = explode(',', $cuePoint->getTags());
+            $containsImportantTag = false;
+            foreach ($importantCuePointTags as $importantCuePointTag)
+            {
+                if (in_array($importantCuePointTag, $cuePointTags))
+                {
+                    $containsImportantTag = true;
+                    break;
+                }
+            }
+            if (!$containsImportantTag)
+            {
+                $cuePoint->setStatus(CuePointStatus::DELETED);
+                $cuePoint->save();
+            }
+        }
+        $vodEntry = entryPeer::retrieveByPK($liveEntry->getRecordedEntryId());
+        if (!$vodEntry)
+        {
+            KalturaLog::err("Recorded entry id [" . $liveEntry->getRecordedEntryId() . "] not found");
+            return null;
+        }
+        $vodEntry->setName($liveEntry->getName() . "_archive");
```

Comment:
new name should be "$liveEntry->getName() <dateString>" 
dateString = currentMonth-3-letter-format date-of-month year-4-digits